src/node/install.sh

```diff
@@ -379,7 +387,7 @@ else
             [ ! -z "$http_proxy" ] && npm set proxy="$http_proxy"
             [ ! -z "$https_proxy" ] && npm set https-proxy="$https_proxy"
             [ ! -z "$no_proxy" ] && npm set noproxy="$no_proxy"
-            npm install -g pnpm
+            npm install -g @pnpm/exe
```

Comment:
I think this is updated to fix the tests? However, both of these packages are different. 

Latest version of `pnpm` is not supported for node 16. Then, only for node 16, can we condition to install `pnpm 7` ? Then keep the rest as is! 

See https://pnpm.io/installation#compatibility

Reply:
Done